WordPressKit/AccountServiceRemoteREST+SocialService.swift

```diff
@@ -13,16 +13,14 @@ extension AccountServiceRemoteREST {
     /// - Parameters:
     ///     - service The name of the social service.
     ///     - token The OpenID Connect (JWT) ID token identifying the user on the social service.
-    ///     - appleEmail The email address associated with an Apple ID. Optional.
-    ///     - appleFullName The full name associated with an Apple ID. Optional.
+    ///     - appleConnectParameters Dictionary containing endpoint parameters specific to connecting to Apple service.
     ///     - oAuthClientID The WPCOM REST API client ID.
     ///     - oAuthClientSecret The WPCOM REST API client secret.
     ///     - success The block that will be executed on success.
     ///     - failure The block that will be executed on failure.
     public func connectToSocialService(_ service: SocialServiceName,
                                        serviceIDToken token: String,
-                                       appleEmail: String? = nil,
-                                       appleFullName: String? = nil,
+                                       appleConnectParameters: [String:AnyObject]? = nil,
```

Comment:
So could we just name this `connectParameters`? Perhaps the doc comment could say "currently only used for the Apple service".

Reply:
Sure.